extensions/replication/tasks/ReplicateObject.js

```diff
@@ -719,17 +722,17 @@ class ReplicateObject extends BackbeatTask {
         this.backbeatSourceProxy.setBackbeatClient(this.backbeatSource);
     }
 
-    _setupDestClients(targetRole, log) {
-        this.destBackbeatHost = this.destHosts.pickHost();
-
-        if (this.destConfig.auth.type === authTypeAssumeRole) {
-            const accountId = _extractAccountIdFromRole(targetRole);
-            const roleName = _extractRoleNameFromRole(targetRole);
-            this.clientManager = new ClientManager({
-                id: accountId,
+    _setupAssumeRoleDestClient(targetRole) {
```

Comment:
Due to how errors are reported as null in case of issue, should we at least add some logs?